pyscf/adc/uadc.py

```diff
@@ -547,6 +576,68 @@ def compute_energy(myadc, t1, t2, eris):
     return e_corr
 
 
+def contract_ladder_outcore_antisym(myadc,t_amp,eris,spin=None):
+
+    v_list = []
+
+    if spin == "alpha":
```

Comment:
Do we have to use the spin argument? It seems that this is something we should be able to get from dimensions of t_amp. Is it correct? We should make sure we do not have redundant arguments. Also, setting spin as a conditional argument, but then adding an exceptions for it is not a good idea. Essentially, this mean that spin is not a conditional argument. So if we had to use it, you would need to set spin to "alpha" as a default value.